### DISCUSSION TITLE
Add timestamp in MQTT shower message

### DISCUSSION THREAD

**Discussion by sciurius** [emsesp/EMS-ESP32#1328] [Ideas]
The shower MQTT message is `{"duration":XXX}`. Would it be possible to add the time stamp?
In HA, the `last-updated` property is not very stable.

**Comment by MichaelDvP** [comment]
If the esp32-RTC isproper set by NTP, thermostat or manual we can add it.
Do you need timestamp or a time-string (or both)? Of shower end or start/end?
e.g.
```
{"duration":600, 
"timestamp": 1697294490,
"time":"2023-10-14T16:42:07"
}
```

**Reply by proddy** [reply]
created an enhancement https://github.com/emsesp/EMS-ESP32/issues/1329
